content/participants/ttjc@mudit999/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Mudit Garg
+type: "participant"
+name: "Mudit Garg"
+portfolio: "I will create one and then update here"
+github-link: "https://github.com/mudit999"
+linkedin-link: "https://www.linkedin.com/in/garg-mudit/"
+looking-for: "job"
+description: "Computer Science | UPES,Dehradun | Beginner - Learning JavaScript, React.js"
+---
+
+Hello! My name is Mudit Garg
+
+### introduction
+
+I am 3 Year B.Tech student. Pursing B.Tech from UPES, Dehradun. I am a Beginner. I have worked a little bit in HTML,CSS, JavaScript but completely new to  Node.js, React.js 
+
+## why are you taking this challenge?
+
+I need an Job in May-June 2021.
```

Comment:
a* Job

Reply:
@khusharth 
Updated my application!!
Thank you :)